include/bluetooth/mesh/access.h

```diff
@@ -56,14 +56,21 @@ extern "C" {
 #define BT_MESH_IS_DEV_KEY(key) (key == BT_MESH_KEY_DEV_LOCAL || \
 				 key == BT_MESH_KEY_DEV_REMOTE)
 
-/** Maximum payload size of an access message (in octets). */
+/** Maximum payload size of an segment access message (in octets). */
 #define BT_MESH_APP_SEG_SDU_MAX   12
+
+/** Maximum payload size of an unsegment access message (in octets). */
+#define BT_MESH_SDU_UNSEG_MAX     11
+
 /** Maximum possible payload size of an outgoing access message (in octets). */
-#define BT_MESH_TX_SDU_MAX        (CONFIG_BT_MESH_TX_SEG_MAX * \
-				  BT_MESH_APP_SEG_SDU_MAX)
+#define BT_MESH_TX_SDU_MAX        MAX((CONFIG_BT_MESH_TX_SEG_MAX * \
+					BT_MESH_APP_SEG_SDU_MAX),  \
+				       (BT_MESH_SDU_UNSEG_MAX + 4))
```

Comment:
Why `BT_MESH_SDU_UNSEG_MAX + 4`? maximum transport control payload is 11 (it doesn't use TransMIC).
Access layer will have `BT_MESH_SDU_UNSEG_MAX - 4`.

Reply:
> Why `BT_MESH_SDU_UNSEG_MAX + 4`? 

Please ref Mesh Profile 1.0.1 3.7.3 Access payload Table 3.42: Maximum useful access payload sizes for various sizes of TransMIC

> maximum transport control payload is 11 (it doesn't use TransMIC).

This macro `BT_MESH_TX_SDU_MAX` not used for transport control message.